docs/source/60_building_apps.md

```diff
@@ -249,3 +252,32 @@ $ spark-submit \
 
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
+
+### Writing your own registrator that calls the ADAM registrator {#registrator}
+
+As we do in ADAM, you may want to provide your own Kryo serializer registrator,
```

Comment:
I hadn't been using "you" in this doc, so this paragraph doesn't use the same voice as above.  I guess I didn't check the other doc pages to see.